src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -31,7 +31,8 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(@RequestParam Long userId,
```

Comment:
README - 2. Replace all ?userId @RequestParam in your code with Authentication object.

Reply:
I didn't catch what I have to fix here. Have I replaced @RequestParam Long movieSessionId with something another?